packages/models/src/resource-binding/types.ts

```diff
@@ -1,4 +1,8 @@
-import { Hash } from '@ckb-lumos/base'
+import { Cell, Hash } from '@ckb-lumos/base'
 
 export type TypeScriptHash = Hash | 'null'
 export type LockScriptHash = Hash
+
+export type RegistryStatus = 'registered' | 'initiated'
+
+export type CellChangeData = [Cell, string]
```

Comment:
Use **labeled tuple** for readability

Ref: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html#labeled-tuple-elements